src/redux/books/book.js

```diff
@@ -0,0 +1,40 @@
+const ADD_A_BOOK = 'ADD_A_BOOK';
+const REMOVE_A_BOOK = 'REMOVE_A_BOOK';
+const LIST_BOOKS = 'LIST_BOOKS';
```

Comment:
- [ ] Following the ducks pattern, your action types should be in the form `app/reducer/ACTION_TYPE` e.g `bookstore/books/ADD_A_BOOK`. Please make modification 👍